extensions/amp-analytics/0.1/visibility-model.js

```diff
@@ -346,13 +298,16 @@ export class VisibilityModel {
     }
     const conditionsMet = this.updateCounters_(visibility);
     if (conditionsMet) {
-      if (this.scheduledRunId_) {
-        clearTimeout(this.scheduledRunId_);
-        this.scheduledRunId_ = null;
+      if (this.scheduledUpdateTimeoutId_) {
+        clearTimeout(this.scheduledUpdateTimeoutId_);
+        this.scheduledUpdateTimeoutId_ = null;
       }
       if (this.reportReady_) {
         this.eventResolver_();
```

Comment:
I believe `this.eventResolver_` was introduced to prevent double triggering.
Since now we officially start to support that, we will not need it anymore. 

we can directly call `this.onTriggerObservable_.fire()`

Reply:
Can do. Could that be in a separate PR, though? It seems like an efficiency improvement rather than functionality, and customer was told they would receive the functionality yesterday, correct?

Reply:
I'm fine with a TODO.

Reply:
Thanks! Added TODO.